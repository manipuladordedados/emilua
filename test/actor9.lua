local inbox = require('inbox')

inbox:close()
inbox:receive()
