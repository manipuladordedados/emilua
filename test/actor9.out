Main fiber from VM 0x1 panicked: 'Channel closed'
stack traceback:
	[string "?"]: in function 'receive'
	input.lua:4: in main chunk
	[C]: in function ''
	[string "?"]: in function <[string "?"]:0>
